src/lib/OpenEXRCore/chunk.c

```diff
@@ -198,7 +201,7 @@ exr_read_scanline_chunk_info (
     int32_t          data[3];
     int64_t          ddata[3];
     int64_t          fsize;
-    uint64_t         dataoff;
+    uint64_t         chunkmin, dataoff;
```

Comment:
This is technically a change to the public ABI, right?

Reply:
how so? it's an internal member and static function?